src/Compilers/Core/Portable/SymbolDisplay/AbstractSymbolDisplayVisitor.cs

```diff
@@ -12,18 +12,21 @@ internal abstract partial class AbstractSymbolDisplayVisitor : SymbolVisitor
         protected readonly ArrayBuilder<SymbolDisplayPart> builder;
         protected readonly SymbolDisplayFormat format;
         protected readonly bool isFirstSymbolVisited;
+        protected readonly bool inTypeArguments;
```

Comment:
>inTypeArguments [](start = 32, length = 15)

"inNamespaceOrType" ?

Reply:
Please rename the field and parameters

Reply:
@AlekseyTs the field indicates if it is in type argument context. I think it is more relevant name.

Reply:
It doesn't have to be that specific, I can imagine using this kind of visitor in other contexts in the future. For example, when we are formatting parameter types withing method signature, etc.